src/find-git-exec.ts

```diff
@@ -73,13 +77,11 @@ function findSpecificGit(path: string, onLookup: (path: string) => void): Promis
 
 async function exec(path: string, command: string | string[]): Promise<string> {
     return new Promise<string>((c, e) => {
-        cp.execFile(path, Array.isArray(command) ? command : [command], { encoding: 'utf-8' }, (error, stdout, stderr) => {
-            if (error) {
-                e(new Error(`Git not found under '${path}'.`));
-                return;
-            }
-            c(stdout.toString().trim());
-        });
+        const buffers: Buffer[] = [];
+        const child = cp.spawn(path, Array.isArray(command) ? command : [command]);
+        child.stdout.on('data', (b: Buffer) => buffers.push(b));
+        child.on('error', e);
+        child.on('close', code => code ? e(new Error(`Git not found under '${path}'.`)) : c(toUtf8String(buffers)));
```

Comment:
`close` guarantees the stdout is finished reading. `execFile` uses this internally as well.